lidarrmetadata/chart.py

```diff
@@ -0,0 +1,171 @@
+"""
+Code for getting and parsing music charts (Billboard, itunes, etc)
+"""
+
+import billboard
+import pylast
+import requests
+
+from lidarrmetadata import config
+from lidarrmetadata import provider
+
+
+def get_apple_music_chart(count=10):
+    """
+    Gets and parses itunes chart
+    :param count: Number of results to return. Defaults to 10
+    :return: Chart response for itunes
+    """
+    URL = 'https://rss.itunes.apple.com/api/v1/us/apple-music/top-albums/all/{count}/explicit.json'.format(count=4 * count)
```

Comment:
How does this work? Are we pulling  4times the amount of albums needed so that we can return the correct count Incase some don’t map?

Reply:
Yeah that's the idea. I don't particularly like it, but I figured it would be better to be consistent that we're returning a number that agrees with the count parameter.